core/src/main/java/org/jruby/util/io/ChannelFD.java

```diff
@@ -131,7 +130,7 @@ private void initChannelTypes() {
         else chRead = null;
         if (ch instanceof WritableByteChannel) chWrite = (WritableByteChannel)ch;
         else chWrite = null;
-        if (ch instanceof SeekableByteChannel) chSeek = (SeekableByteChannel)ch;
```

Comment:
What's the reason for this? SeekableByteChannel is not a new class, is it? I suppose on current Java the only SeekableByteChannel is FileChannel, but I prefer to use the more generic interface if possible.